static/js/place/place_template.tsx

```diff
@@ -532,27 +532,25 @@ class ChartBlock extends Component<ChartBlockPropType, unknown> {
           this.props.config
         );
     return (
-      <div className="chart-block">
-        <div className="row row-cols-md-2 row-cols-1">
-          {configList.map((item) => {
-            const id = randDomId();
-            return (
-              <Chart
-                key={id}
-                id={id}
-                config={item}
-                dcid={this.props.dcid}
-                placeType={this.props.placeType}
-                parentPlaces={this.props.parentPlaces}
-                childPlaces={this.props.childPlaces}
-                similarPlaces={this.props.similarPlaces}
-                nearbyPlaces={this.props.nearbyPlaces}
-                chartData={this.props.chartData}
-              />
-            );
-          })}
-        </div>
-      </div>
+      <>
```

Comment:
good to know, so this also does not create top level parent element?

Reply:
it's a container react fragment, but it doesn't translate to additional dom nodes. useful for returning lists :)